packages/manager/src/features/CloudPulse/Alerts/NotificationChannels/NotificationsChannelsListing/NotificationChannelTableRow.tsx

```diff
@@ -19,16 +26,31 @@ interface NotificationChannelTableRowProps {
 export const NotificationChannelTableRow = (
   props: NotificationChannelTableRowProps
 ) => {
-  const { notificationChannel } = props;
+  const { handlers, notificationChannel } = props;
   const { data: profile } = useProfile();
-  const { id, label, channel_type, created_by, updated, updated_by, alerts } =
-    notificationChannel;
+  const {
+    id,
+    label,
+    channel_type,
+    created_by,
+    updated,
+    updated_by,
+    alerts,
+    type,
+  } = notificationChannel;
   return (
     <TableRow
       data-qa-notification-channel-cell={id}
       key={`notification-channel-row-${id}`}
     >
-      <TableCell>{label}</TableCell>
+      <TableCell>
+        <Link
+          data-qa-alert-link
+          to={`/alerts/notification-channels/detail/${id}`}
```

Comment:
This is fine, but we can lean into the type-safety of TanStack router here if we want by using the `params` prop

```suggestion
          params={{ channelId: String(id) }}
          to={'/alerts/notification-channels/detail/$channelId'}
```